### ISSUE TITLE
(QUERY): Left vs Right maximize - Gap between window and edge

### ISSUE THREAD

**Issue by hilprl** [raboof/notion#254]
#### DESCRIPTION

I am not sure if the this is an intended behavior or a bug.

I have two frames vertically split and each contain two tabs. After performing a horizontal left maximize of the right frame, there is a noticeable gap between the maximized frame and  the edge. A small portion of underlying window can be seen. 

When a right maximize of left frame is performed, there is no gap between the maximized window and edge
 
* Left Horizontal Maximize *

![left_horiz_expand](https://user-images.githubusercontent.com/7909450/75110897-d37d2d00-562b-11ea-95ef-0ec438bde4e4.png)

* Right Horizontal Maximize *

![right_horiz_expand](https://user-images.githubusercontent.com/7909450/75110903-e42da300-562b-11ea-9e94-4e24e2adf734.png)

**Comment by raboof** [comment]
On master, when I try this, both left-maximize and right-maximize leave a small gap. I think that's nice.

However, the gap is not the same size on each side. That seems inelegant and potentially something worth fixing.

> Is there a way to make left horizontal maximize to behave like this?

You mean not having a gap? I would be fine with having (and documenting) a way to configure notion to behave like this, but I don't think it should be the default.

**Comment by hilprl** [comment]
> On master, when I try this, both left-maximize and right-maximize leave a small gap. I think that's nice.
> 
> However, the gap is not the same size on each side. That seems inelegant and potentially something worth fixing.
> 

Yes, this was the behavior I observed when I raised this ticket. It would be nice if the gaps are of the same size on each side. On my machine, the left gap becomes more evident when number of tabs are >3. Until then it is quite unnoticeable.

> > Is there a way to make left horizontal maximize to behave like this?
> 
> You mean not having a gap? I would be fine with having (and documenting) a way to configure notion to behave like this, but I don't think it should be the default.

I totally agree. Thanks.
